the-collab-lab/tcl-21-smart-shopping-list
# As th add new item

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

1. Created a form in AddItem Component that enables the user to add a new shopping item by name as well as how soon they will purchase it again. 

2. It then saves the item to the database alongside the user's token. 

3. The item is then displayed in the List component if the item token matches the user token. 



## Related Issue



## Acceptance Criteria





## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria



1. Pull down remote branches: git pull
2. Navigate to correct branch: git checkout as-th-add-new-item
3. Run server and view in browser on localhost:3000: npm start
4. Click "Add New Item" to add a new grocery item that will persist to the database
5. Clicking the "List" button should then redirect you to the list view where you can see the item you just added and any others associated with that user token
